src/main/java/data/TaskManager.java

```diff
@@ -214,41 +204,63 @@ public static TaskType parseTaskType(String userInput) {
      * @throws TaskManagerException If not in correct week/month view.
      * @throws DateTimeParseException If there is an error parsing the date.
      */
-    public static void updateManager(Scanner scanner, WeekView weekView, boolean inMonthView,TaskManager taskManager)
+//    public static void updateManager(Scanner scanner, WeekView weekView, boolean inMonthView,TaskManager taskManager)
+//            throws TaskManagerException, DateTimeParseException {
+//        System.out.println("Enter the date for the task you wish to update (dd/MM/yyyy):");
+//        LocalDate date = parseInputDate(scanner);
+//
+//
+//        if (inMonthView) {
+//            checkIfDateInCurrentMonth(date);
+//        } else {
+//            checkIfDateInCurrentWeek(date, weekView);
+//        }
+//
+//        listTasksAtDate(taskManager, date, "Enter the task number of the task you wish to update:");
+//
+//        int taskNumber;
+//        String updatedDescription;
+//
+//        try {
+//            taskNumber = Integer.parseInt(scanner.nextLine().trim());
+//            assert taskNumber != 0 : "Task Number is invalid!";
+//
+//            System.out.println("Enter the updated task description:");
+//            updatedDescription = scanner.nextLine().trim();
+//
+//            updateTask(date, taskNumber - 1, updatedDescription);
+//            saveTasksToFile(tasks,Storage.FILE_PATH); //Update tasks.txt file
+//            System.out.println("Task updated.");
+//        } catch (NumberFormatException e) {
+//            System.out.println("Task number should be an integer value. Please try again.");
+//        } catch (IndexOutOfBoundsException e) {
+//            System.out.println("The task number you have entered does not exist. Please try again.");
+//        }
+//
+//    }
+
+    public static void updateManager(Scanner scanner, WeekView weekView, boolean inMonthView, TaskManager taskManager,
```

Comment:
Comment indentation is causing checkstyle test to fail. Also large segments of commented out code is bad coding practice. Do rectify